.travis.yml

```diff
@@ -0,0 +1,36 @@
+env:
+  global:
+    - CC_TEST_REPORTER_ID=64d1468e17f8994daaac252fab3e9eacc7fc6c58366069a32473d21d2d684f24
+language: node_js
+node_js:
+  - "8"
+  - "6"
+  - "10"
+os:
+  - linux
+  - osx
+cache:
+  directories:
+  - node_modules # NPM packages
+before_script:
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+      sudo apt-get install clang-format;
+      curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter;
+    elif [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+      brew install clang-format;
+      curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-darwin-amd64 > ./cc-test-reporter;
+    else
+      echo Error:TRAVIS_OS_NAME && exit 1;
```

Comment:
Should `Error:TRAVIS_OS_NAME` be using the environment variable like`$TRAVIS_OS_NAME`?

Reply:
I went with what Atom Beautify has: https://github.com/Glavin001/atom-beautify/blob/master/.travis.yml#L76